### PR TITLE
doc(ja): translate `credits` into Japanese

### PR DESCRIPTION
**Pull request by hitohata**
## Summary

I've translated the credits page. 

Related to: #880

### CODE REVIEW

**Comment by nissy-dev** [comment]
I'll review

File: website/src/content/docs/ja/internals/credits.mdx [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+title: クレジット
+description: これまでのコントリビューターと私達に影響を与えたプロジェクトへの謝辞
```
Comment (nissy-dev) [inline website/src/content/docs/ja/internals/credits.mdx:3]:
```suggestion
description: これまでのコントリビューターと Biome に影響を与えたプロジェクトへの謝辞
```

File: website/src/content/docs/ja/internals/credits.mdx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+<Contributors />
+
+## 謝辞
+
+Biomeには他のプロジェクトから多大な影響を受けたコードが含まれています。それらはBiomeの言語/インフラストラクチャに適応されています。
+
+- [Prettier](https://github.com/prettier/prettier/)
+  - [ライセンス](https://github.com/biomejs/biome/blob/main/crates/biome_js_formatter/LICENSE)
+
+## フォーク
```
Comment (nissy-dev) [inline website/src/content/docs/ja/internals/credits.mdx:17]:
```suggestion
## フォークしたプロジェクト
```

File: website/src/content/docs/ja/internals/credits.mdx [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+Biomeは[Rome Tools](https://github.com/biomejs/biome/)のコミュニティフォークです。
```
Comment (nissy-dev) [inline website/src/content/docs/ja/internals/credits.mdx:19]:
```suggestion
Biomeは、[Rome Tools](https://github.com/biomejs/biome/)のコミュニティによるフォークです。
```

File: website/src/content/docs/ja/internals/credits.mdx [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+_Rome Tools_ と _Biome_ はMITライセンスに基づいています。
```
Comment (nissy-dev) [inline website/src/content/docs/ja/internals/credits.mdx:20]:
```suggestion
_Rome Tools_ と _Biome_ は、MITライセンスに基づいています。
```

File: website/src/content/docs/ja/internals/credits.mdx [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+
+Biomeには他のプロジェクトからフォークしたコードが含まれています。
+それらのコードは何らかの方法にて変換され、ときには大幅に書き直されています。
```
Comment (nissy-dev) [inline website/src/content/docs/ja/internals/credits.mdx:24]:
```suggestion
それらのコードは、自分たちのコードに合うように変換されたり、ときには大幅に書き直されたりしています。
```

File: website/src/content/docs/ja/internals/credits.mdx [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+- [Prettier](https://github.com/prettier/prettier/)
+  - [ライセンス](https://github.com/biomejs/biome/blob/main/crates/biome_js_formatter/LICENSE)
+
+## フォーク
+
+Biomeは[Rome Tools](https://github.com/biomejs/biome/)のコミュニティフォークです。
+_Rome Tools_ と _Biome_ はMITライセンスに基づいています。
+
+
+Biomeには他のプロジェクトからフォークしたコードが含まれています。
```
Comment (nissy-dev) [inline website/src/content/docs/ja/internals/credits.mdx:23]:
```suggestion
Biomeには、他のプロジェクトからフォークしたコードが含まれています。
```

File: website/src/content/docs/ja/internals/credits.mdx [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+description: これまでのコントリビューターと私達に影響を与えたプロジェクトへの謝辞
+---
+
+import Contributors from "@src/components/Contributors.astro";
+
+<Contributors />
+
+## 謝辞
+
+Biomeには他のプロジェクトから多大な影響を受けたコードが含まれています。それらはBiomeの言語/インフラストラクチャに適応されています。
```
Comment (nissy-dev) [inline website/src/content/docs/ja/internals/credits.mdx:12]:
```suggestion
Biomeには、他のプロジェクトから大きな影響を受けたコードが含まれています。それらは、Biomeのインフラストラクチャに適応されています。
```

**Review by nissy-dev — CHANGES_REQUESTED** [review]
翻訳ありがとうございます！コメントしたところは commit suggestion ボタンで修正してもらえると助かります 🙏

**Review by nissy-dev — APPROVED** [review]
Thank you for your contribution!

**Comment by hitohata** [comment]
Sorry for the late reply and action...
Thank you for merging!

**Pull request merged**